### PR TITLE
fix CDN issue

### PR DESCRIPTION
**Pull request by csmengwan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 
### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).
### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [ ] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---

This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/632)

### CODE REVIEW

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 1]
```diff
@@ -1750,11 +1750,19 @@
         },
         "httpPort": {
           "description": "The value of the HTTP port. Must be between 1 and 65535",
-          "type": "integer"
+          "type": "integer",
+		  "maximum": 65535,
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: indentation 2 spaces

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 2]
```diff
@@ -1750,11 +1750,19 @@
...
+		  "exclusiveMaximum": false,
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: false value for exclusiveMaximum and exclusiveMinimum is redundant.

The spec says:
> if "exclusiveMaximum" is **not present, or has boolean value false**, then the instance is valid if it is lower than, or equal to, the value of "maximum";

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 3]
```diff
@@ -1750,11 +1750,19 @@
...
+		  "minimum": 1,
+		  "exclusiveMinimum": false
         },
         "httpsPort": {
           "description": "The value of the HTTPS port. Must be between 1 and 65535",
-          "type": "integer"
+          "type": "integer",
+		  "maximum": 65535,
+		  "exclusiveMaximum": false,
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: same as above

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 4]
```diff
@@ -1843,11 +1851,19 @@
         },
         "httpPort": {
           "description": "The value of the HTTP port. Must be between 1 and 65535.",
-          "type": "integer"
+          "type": "integer",
+		  "maximum": 65535,
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: same as above

**Comment by hovsepm** [comment]
LGTM

**Comment by amarzavery** [comment]
@csmengwan - Can you relove Hovsep's comment:

> DeepCreatedOrigin should be removed completely. Instead Origin model should be used.

**Comment by hovsepm** [comment]
removing DeepCreatedOrigin is not that easy. we need to rethink this.

**Comment by hovsepm** [comment]
LGTM

**Pull request merged**